tsconfig.json

```diff
@@ -1,15 +1,20 @@
 {
   "compilerOptions": {
+    "outDir": "./dist",
+    "declarationDir": "./dts",
+    "target": "es6",
```

Comment:
Hmm... Shouldn't the target be es5? Some of the browsers we need to support can't handle es6 yet.